### PR TITLE
Adds a checker on PHP namespace and fixes them

### PR DESCRIPTION
**Pull request by BitOne**
**Description**

This PR adds a static check on PHP `use` existence.

This allows to quickly check if a class, interface or trait mentioned in documentation code sample still exists.

**Definition Of Done**

| Q                                 | A
| --------------------------------- | ---
| Technical Review and 2 GTM        | Todo
| English Review and 1 GTM          | Todo

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

**Comment by jmleroux** [comment]
Interesting idea ! :+1: 
(CLA check btw)

File: scripts/check_uses_existence.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+$errorsCount = 0;
+$usesCount = 0;
+
+while ($use = fgets($usesList)) {
+    $useInfo = explode('#', $use);
+
+    $lineLocation = $useInfo[0];
+    $useValue = trim($useInfo[1]);
+
+    if (!class_exists($useValue)) {
```
Comment (jmleroux) [inline scripts/check_uses_existence.php:16]: We can use `&&` instead of 3 `if`

File: scripts/test_php_uses.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+# Extract "use" statement from php and rst files
+# Acme\CustomBundle and Acme\RangeBundle are excluded because created in the examples
+find . \( -iname "*.rst" -or -iname "*.php" \) -not -path "./ee_dev/*" -not -path "./.composer/*" | \
+    xargs grep -ne '^ *use.*\\.*;$' | \
+    sed -e 's/ *use */#/' | \
+    grep -v 'CustomBundle' | \
+    grep -v 'RangeBundle' | \
+    cut -d " " -f 1 | \
+    tr -d ";" > $EE_STD_DIR/php_uses.list
```
Comment (jmleroux) [inline scripts/test_php_uses.sh:22]: Such a beautiful find! :smile:

File: scripts/check_uses_existence.php (hunk 1 above)
Comment (BitOne) [reply scripts/check_uses_existence.php]: Yes, thanks!

File: scripts/check_uses_existence.php [hunk 3]
```diff
@@ -13,13 +13,12 @@
...
-    if (!class_exists($useValue)) {
-        if (!interface_exists($useValue)) {
-            if (!trait_exists($useValue)) {
-                $errorsCount++;
-                printf("Unable to find class or interface or trait %s (defined in %s)\n", $useValue, $lineLocation);
-            }
-        }
+    if ((!class_exists($useValue)) &&
+        (!interface_exists($useValue)) &&
+        (!trait_exists($useValue))) {
```
Comment (jmleroux) [inline scripts/check_uses_existence.php:18]:
```suggestion
    if (!class_exists($useValue) &&
        !interface_exists($useValue) &&
        !trait_exists($useValue)) {
```

**Pull request merged**